app/controllers/hipsters/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Hipsters::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+
+
+ def facebook
+    @user = Hipster.from_omniauth(request.env["omniauth.auth"])
+    if @user.persisted?
+      sign_in_and_redirect @user, :event => :authentication
+      set_flash_message(:notice, :success, :kind => "Facebook") if is_navigation_format?
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)